### ISSUE TITLE
Schema verification of inline !vault results in "unknown tag"

### ISSUE THREAD

**Issue by petercrocker** [ansible-community/vscode-ansible#103]
Issue: `!vault |` results in `unknown tag`

Encrypting individual variables in an inventory results in schema errors. Relevant docs for ansible vault:

https://docs.ansible.com/ansible/latest/user_guide/vault.html#encrypting-individual-variables-with-ansible-vault

Here's an anonymized snippet from my ansible inventory:

```
snmp:
  addresses:
    - "10.44.16.10"
  rocommunity: !vault |
    $ANSIBLE_VAULT;1.1;AES256
    31353063366434366235343939393130626666383561336638613963633636393962656231646536
    3237316430363038303632383464653432336534666336340a333262653237396137623564343630
    38656561323031356265653964653238633636393832643062356338666538303431333337653033
    3836666439393132300a383633376465366130356237633439323639353335383338333465343935
    33663861333232663865316237656138373133393461333733313639383733391234

```

And screenshot from VS Code ansible extension showing issue (https://github.com/ansible-community/vscode-ansible):

![Screenshot 2021-05-07 at 11 07 25](https://user-images.githubusercontent.com/4776300/117434401-8a5d8400-af24-11eb-89b5-cc0700d5a183.png)

**Comment by JPinkney** [comment]
Can you try adding `!vault` to the yaml.customTags setting in vscode like:
```
 "yaml.customTags": [
    "!vault"
 ]
```

**Comment by petercrocker** [comment]
> Can you try adding `!vault` to the yaml.customTags setting in vscode like:
> 
> ```
>  "yaml.customTags": [
>     "!vault"
>  ]
> ```

Works like a charm! Thanks for the workaround.

**Comment by FlorianLaunay** [comment]
Needed tags are now automatically added : https://github.com/ansible-community/vscode-ansible/pull/104

@ssbarnea do we close this issue ?

**Comment by ssbarnea** [comment]
Fixed in 0.4.3 which adds these automatically.
